src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd

```diff
@@ -1844,6 +1844,11 @@ elementFormDefault="qualified">
             <xs:documentation><!-- _locID_text="EnableDefaultItems" _locComment="" -->Defaults to true, and if set to false will disable all default item globs.</xs:documentation>
         </xs:annotation>
     </xs:element>
+    <xs:element name="IsAotCompatible" type="msb:boolean" substitutionGroup="msb:Property">
+        <xs:annotation>
+            <xs:documentation><!-- _locID_text="IsAotCompatible" _locComment="" -->Indicates whether a class library is compatible with native AOT. Setting to true will enable trimming, single file, and AOT analyzers.</xs:documentation>
```

Comment:
> Setting to true will enable trimming, single file, and AOT analyzers.

At first read this makes it sound to me like it's turning on trimming, it's turning on single file publishing, and it's turning on AOT analyzers. I think it'd be clearer instead like:
```suggestion
            <xs:documentation><!-- _locID_text="IsAotCompatible" _locComment="" -->Indicates whether a class library is compatible with native AOT. Setting to true will enable analyzers for trimming, single file, and AOT.</xs:documentation>
```